ClassicStartLoc/StartMenuL10N.ini

```diff
@@ -30,7 +30,7 @@ Menu.NoMatch = لا توجد أية عناصر تطابق البحث.
 Menu.MoreResults = الاطلاع على مزيد من النتائج
 Menu.Help = التع&ليمات والدعم
 Menu.Run = تش&غيل...
-Menu.Logoff = ت&سجيل خروج %s‎...
+Menu.Logoff = ت&سجيل خروج %s.
```

Comment:
Okay I'll fix that tomorrow. I'm clearly too tired for this.

Reply:
👍 👍 👍